frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -123,13 +123,13 @@ public void test()
         testApiDescription(channel, listInferenceApisResult);
         testDescribeApi(channel);
         testUnregisterModel(managementChannel, "noop", null);
-        testLoadModel(managementChannel, "noop.mar", "noop_v1.0");
+        testLoadModel(managementChannel, "noop.mar", "noop_v1.0", "1.11");
         testSyncScaleModel(managementChannel, "noop_v1.0", null);
         testListModels(managementChannel);
         testDescribeModel(managementChannel, "noop_v1.0", null, "1.11");
-        testLoadModelWithInitialWorkers(managementChannel, "noop.mar", "noop");
-        testLoadModelWithInitialWorkers(managementChannel, "noop.mar", "noopversioned");
-        testLoadModelWithInitialWorkers(managementChannel, "noop_v2.mar", "noopversioned");
+        testLoadModelWithInitialWorkers(managementChannel, "noop.mar", "noop", "1.11");
```

Comment:
remove the versions from the UT. This should not be required while loading a model.